### ISSUE TITLE
Drop "raw_images" input type

### ISSUE THREAD

**Issue by joeyparrish** [google/shaka-streamer#25]
Now that we have "external_command" as an input type, it's no longer clear if "raw_images" offers anything unique or valuable.

Thoughts?  Please comment!

If nobody has an opinion on this, I may drop it soon.

**Comment by Romantic-LiXuefeng** [comment]
I'm not clear what is the use case for "external_command" as an input type?

**Comment by joeyparrish** [comment]
It allows you to run an external command which generates content.  For example, we use it to generate a sequence of images with an external command called [Gource](https://gource.io).  That is how we generate this live stream:

https://nightly-dot-shaka-player-demo.appspot.com/demo/#asset=https://storage.googleapis.com/shaka-live-assets/player-source.mpd

Before we added "external_command", we used to do this with "raw_images" through a fifo.  But that required a separate fifo and an additional script to manage it.

Also, "external_command" is more flexible because it can generate an image sequence, or video, or audio, whereas "raw_images" requires a sequence of images specifically.

So I'm struggling to come up with a good reason to keep "raw_images".
